src/types.ts

```diff
@@ -53,6 +54,8 @@ export type BaseMethodArgs = {
   token?: string;
 };
 
+export type UpdateMethodArgs = BaseMethodArgs & TriggerIdType;
```

Comment:
Why do we want to export this one from our main `types` file? How are you expecting developers to use this type?